### PR TITLE
cleanups, repo tweaks, drone manifest

### PR DESCRIPTION
**Pull request by Maciej-Gierwatowski**

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -38,21 +58,30 @@ Use this plugin for deploying a docker container application to AWS EC2 Containe
...
+* `task_kill_on_timeout` - When task reaches timeout - send kill signal to the containers. Deafult `true`
 
 
 ## Example
 
 ```yaml
 steps:
   - name: Deploy to ECS
-    image: pelotech/drone-ecs
+    image: ////
```
Comment (wszychta) [inline DOCS.md:79]: I suspect that this will change to proper plugin image path

File: build.sh [hunk 2]
```diff
@@ -20,4 +20,4 @@ go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/a
 
 
 ## Build docker image
-#docker build  --no-cache --rm --pull -f Dockerfile -t "drone-standalone-tasks:0.0.2" .
+#docker build  --no-cache --rm --pull -f Dockerfile -t "drone-standalone-task:0.0.2" .
```
Comment (wszychta) [inline build.sh]: Mayby we should remove this commented lines?

**Review by bmkrra — APPROVED** [review]

File: main.go [hunk 3]
```diff
@@ -292,6 +322,16 @@ func run(c *cli.Context) error {
 		Volumes:                      c.StringSlice("volumes"),
 		EfsVolumes:                   c.StringSlice("efs-volumes"),
 		PlacementConstraints:         c.String("placement-constraints"),
+
+		CapacityProviders:            c.StringSlice("capacity-providers"),
+        EnableExecuteCommand:         c.Bool("enable-execute-command"),
```
Comment (wszychta) [inline main.go]: Github formating or something else?

File: plugin.go [hunk 4]
```diff
@@ -398,37 +412,63 @@ func (p *Plugin) Exec() error {
...
+	    cap := &ecs.CapacityProviderStrategyItem {
+	        Base:               &base,
+	        Weight:             &weight,
+	        CapacityProvider:   &parts[2],
+	    }
+	    taskParams.CapacityProviderStrategy = append(taskParams.CapacityProviderStrategy, cap)
     }
-    //fmt.Println(taskParams)
+
+    // Standalone task's placementConstraints seems to be redundant if we already use them in task definition... yes/no?
```
Comment (wszychta) [inline plugin.go:465]: Remove it if not in use?

File: DOCS.md (hunk 1 above)
Comment (Maciej-Gierwatowski) [reply DOCS.md:79]: Actually don't have idea for now what to put here. The plugin isn't supposed to be on dockerhub at least for now. And I don't want to put ECR url in public docs.

File: build.sh (hunk 2 above)
Comment (Maciej-Gierwatowski) [reply build.sh]: Actually whole script was for my local tests ;) 
I'd rather gitignore it

File: plugin.go (hunk 4 above)
Comment (Maciej-Gierwatowski) [reply plugin.go:465]: I'd leave this comment to keep it in back of the head. And we'll see whether it's needed when we go live with this plugin.

File: main.go (hunk 3 above)
Comment (Maciej-Gierwatowski) [reply main.go]: fixed

**Review by wszychta — APPROVED** [review]
LGTM

**Pull request merged**